### ISSUE TITLE
VS crashes instantly after installing this extension

### ISSUE THREAD

**Issue by kfrancis** [adospace/vs-android-emulator#1]
I formatted my machine recently, but I've been doing xamarin/maui dev since. It's been working fine, but after installing the extension - it crashes on the load of the app.

- VS 2022 v17.7.0
- VS 2022 Preview (v17.8.0 Preview 1.0)
Stack from event viewer:

```
Application: devenv.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.DirectoryNotFoundException
   at System.IO.__Error.WinIOError(Int32, System.String)
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].CommonInit()
   at System.IO.FileSystemEnumerableIterator`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]..ctor(System.String, System.String, System.String, System.IO.SearchOption, System.IO.SearchResultHandler`1<System.__Canon>, Boolean)
   at System.IO.Directory.GetFiles(System.String, System.String)
   at VsAndroidEm.EmulatorControlViewModel.Timer_Tick(System.Object, System.EventArgs)
   at System.Windows.Threading.DispatcherTimer.FireTick(System.Object)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
```

Attached is the activity log, which shows it's an extension crash that's occurring in the EmulatorControlViewModel Timer_Tick:
![2023-08-14 14 09 14](https://github.com/adospace/vs-android-emulator/assets/67574/627fa5ba-3efd-4764-b3ad-9c13464f90e4), which must be this code: https://github.com/adospace/vs-android-emulator/blob/main/src/EmulatorControlViewModel.cs#L66-L67

[ActivityLog.txt](https://github.com/adospace/vs-android-emulator/files/12338131/ActivityLog.txt)

**Comment by kfrancis** [comment]
I was able to revert for now using this (in dev command prompt), though I'd like to definitely use this when it works:

`VSIXInstaller.exe /uninstall:VsAndroidEm.5dfcfa73-0d82-4b24-9c01-479eaa271e61`

**Comment by adospace** [comment]
Hi, thank you for the detailed issue description. Yes, that's the problem, the extension monitor this folder to know when an emulator is running:
C:\Users\user\AppData\Local\Temp\avd\running

In your case, that folder doesn't exist (yet?). 
Please download the latest version of the extension just published (1.6) which should at least not crash the IDE.

**Comment by kfrancis** [comment]
Can confirm, that's now working with that change. Thank you for the fix!
